v2/.gitattributes

```diff
@@ -0,0 +1,18 @@
+# ~~ Generated by projen. To modify, edit .projenrc.js and run "npx projen".
```

Comment:
same here?

Reply:
unfortunately this is managed by projen, so it needs to exist in both folders. I'll investigate `.eslintrc.json` though